lib/general.js

```diff
@@ -49,8 +49,15 @@ class OptionalOpencv4nodejsCommandCheck extends DoctorCheck {
     let stdout = '';
     const packageName = 'opencv4nodejs';
 
+    const npmPath = await resolveExecutablePath('npm');
```

Comment:
this won't help, since on windows the script is called npm.cmd. Try to add this suffix if Windows OS is detected like it is done in `getBinaryNameForOS` method inside appium-adb

Reply:
ah...

Reply:
https://github.com/appium/appium-support/blob/master/lib/system.js#L18